flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/opendj/values.yaml

```diff
@@ -28,7 +28,7 @@ image:
   # -- Image  to use for deploying.
   repository: gluufederation/opendj
   # -- Image  tag to use for deploying.
-  tag: 5.0.0-12
+  tag: 5.0.0_dev
```

Comment:
set the tag to non-dev version and let users override the tag via top-level values.yaml

Reply:
Since there's only dev image tag right now for opendj, we can keep the dev tag here.